ros/api/v1/recommendations.py

```diff
@@ -53,7 +54,25 @@ def get(self, host_id):
                 if rule_data:
                     rule_dict = rule_data.__dict__
                     recommendation = {}
+                    instance_price = 0
+                    summary = ''
+                    candidate_string = ''
+                    rule_hit_details = rule_hit.get('details')
+                    candidates = rule_hit_details.get('candidates')
+                    summaries = rule_hit_details.get('summary')
+                    instance_price += rule_hit_details.get('price')
```

Comment:
```suggestion
                    instance_price += instance_price = f'{float(rule_hit_details.get('price'))} {INSTANCE_PRICE_UNIT}'
```

You can keep it here and remove it from contents in seed file.

Reply:
We can also use without float

Reply:
ok